apis/projectcontour/v1alpha1/httploadbalancer.go

```diff
@@ -79,7 +79,7 @@ type TLS struct {
 // Route contains the set of routes for a virtual host
 type Route struct {
 	// Condition defines additional routing parameters on the route
-	Condition `json:"condition,omitempty"`
+	*Condition `json:"condition,omitempty"`
```

Comment:
nit: embedding is cool, but this is probably better written in this instance as
```
type Route struct {

       Condition *Condition
}
```
this will prevent the compiler letting us call Route.(some field on Condition)